index.js

```diff
@@ -19,14 +19,26 @@ function parse(req) {
 
     let baton = {
       label: query.label || ((query.compression ? 'gzip ' : '') + 'size'),
-      color: query.color || 'brightgreen',
+      color: query.color || null,
```

Comment:
If the user does not want the `test` feature, the badge must have a default color.